cmd/generator.go

```diff
@@ -29,6 +29,7 @@ var (
 	additionalQueries      = kingpin.Flag("query-additional-queries", "PromQL queries to run in addition to the default.").Strings()
 	tenantsCount           = kingpin.Flag("tenants-count", "Number of tenants to fake.").Default("1").Int()
 	seriesCount            = kingpin.Flag("series-count", "Number of series to generate for each tenant.").Default("1000").Int()
+	seriesChurnPeriod      = kingpin.Flag("series-churn-period", "How frequently the series should churn. 0 to disable churning.").Default("0").Duration()
```

Comment:
Might be worth explaining here that each series will churn over this duration (ie. they won't churn all at the same time)

Reply:
I attempted it here: https://github.com/pracucci/cortex-load-generator/pull/8/commits/cd3a4ff0cae8058397fc8aca243c8275143e2545